src/WorkerChannel.ts

```diff
@@ -15,8 +15,14 @@ export class WorkerChannel {
     eventStream: IEventStream;
     functionLoader: IFunctionLoader;
     packageJson: PackageJson;
+    hostVersion = '';

```

Comment:
```suggestion
    /**
     * This will only be set after worker init request is received
     */
    hostVersion?: string;
```

Making it optional with `?` is more tedious (you'll have to use `nonNullProp` more), but more correct. Timing issues are prime candidates for `undefined`/`nonNullProp` and this fits the bill because someone might try to access `hostVersion` before we get the worker init request